src/plots/polar/constants.js

```diff
@@ -24,6 +24,9 @@ module.exports = {
         'frontplot',
         'angular-axis',
         'radial-axis',
+        // TODO hmm, putting angular-line above radial-axis
+        // can sometimes hide radial ticks and label
+        // this happens especially for the 'inner' angular axis
```

Comment:
Also I guess if we add shadows around the radial tick labels, like the legacy polar plots have, it would be important to bring these in front of the lines.

Reply:
> are there any downsides to moving angular-line behind radial-axis?

Here are the results:

https://github.com/plotly/plotly.js/compare/polar-hole...polar-lines-below-ticks

I can't detect anything that looks _off_. Note that cartesian subplots also put axis lines below axis ticks:

https://github.com/plotly/plotly.js/blob/907070555924a034a7e3c2052d6fe114144e3215/src/plots/cartesian/index.js#L492-L498

Ok to merge these `polar-lines-below-tick` commits in this PR?

Reply:
I don't detect any changes at all, even if git does. So seems fine, but I'd like to see a mock that's *designed* to change in this case - so, fat lines of weird colors overlapping ticks/labels of unrelated colors - and verify that we like the results

Reply:
With https://github.com/plotly/plotly.js/pull/2977/commits/9c3dcef207184909f5637fbb0721ce7740f9f9bf `polar_hole` now produces this diff

![peek 2018-09-07 17-49](https://user-images.githubusercontent.com/6675409/45244615-98b3f780-b2c6-11e8-8fc4-95c22fe9091e.gif)